### ISSUE TITLE
Turning off logging prevents Windows distribution from launching

### ISSUE THREAD

**Issue by mliberty1** [jetperch/pyjoulescope_ui#120]
Platform: Win 10 x64
Version: 0.9.3 released (works fine unreleased)

**Description**
If you turn off logging, File → Preferences → General → log_level → OFF, then close and relaunch the Joulescope UI, it does not start.  It displays:
```
--------------------
Exception on Joulescope UI 0.9.5 startup. 
Python=3.9.2 (tags/v3.9.2:1a79785, Feb 19 2021, 13:44:55) [MSC v.1928 64 bit (AMD64)]
Platform=Windows-10-10.0.19041-SP0
Traceback (most recent call last):
  File "joulescope_ui\main.py", line 1877, in run
  File "joulescope_ui\logging_util.py", line 167, in logging_config
AttributeError: 'NullWriter' object has no attribute 'fileno'
```

**Expected Behavior**
You should be able to turn off logging and have the Joulescope UI run.

**Comment by mliberty1** [comment]
The cause of the failure is [here](https://github.com/jetperch/pyjoulescope_ui/blob/ee814d50683029238013cc48d9144d8f06449eaa/joulescope_ui/logging_util.py#L167).  It looks like you turned off the Joulescope UI logging, which I suspect most people never touch.  The [faulthandler.enable](https://docs.python.org/3/library/faulthandler.html) function attempts to use stderr by default.  This works fine in development and for every other platform.  In the distributed version on Windows, stderr apparently does not exist, which causes this function to fail.  I successfully duplicated the issue on my Windows machine.

**Workaround**
The workaround is to turn logging back on.  If you don't care about your other Joulescope UI settings, you can open File Explorer, type "%LOCALAPPDATA%\joulescope" into the navigation bar, like this: 
![image](https://user-images.githubusercontent.com/5393422/111832024-9e8ee880-88c6-11eb-8058-4bde743886b5.png)

Then select all four items with your mouse, and press delete. 
![image](https://user-images.githubusercontent.com/5393422/111831968-8323dd80-88c6-11eb-8598-6ad70d6de44d.png)

You should then be able to launch the Joulescope UI and have it run.

If you do care about your settings, navigate into the config directory and open "joulescope_config.json" in your favorite text editor.  Search for "General/log_level" and edit so that they are all set to "INFO".

**Comment by mliberty1** [comment]
Fixed in [v0.9.7](https://github.com/jetperch/pyjoulescope_ui/releases/tag/v0.9.7).
